great_expectations/rule_based_profiler/domain_builder/domain_builder.py

```diff
@@ -144,3 +144,26 @@ def get_batch_id(
             )
 
         return batch_ids[0]
+
+
+def build_domains_from_column_names(column_names: List[str]) -> List[Domain]:
```

Comment:
@anthonyburdi Do we want a column-domain specific logic in the general `DomainBuilder` module, or should it go to `util.py`?  My vote is for the latter.

Reply:
I second @alexsherstinsky's thought here. I think this function would be better-placed in a `util.py`

Reply:
I understand the logic, but I disagree. The rationale is that this functionality is specific and tightly coupled to domains so I would look for it in domain.py rather than a general `util.py` module. That being said, I do agree it should be moved from it's current home in `domain_builder.py` to `domain.py`.